src/assets/style.scss

```diff
@@ -210,4 +210,8 @@ span {
   font-size: 40px;
   height: 100px;
   width: 100px;
+}
+
+.vjs-tree {
+  word-wrap: normal !important;
```

Comment:
Good idea! 

Although I feel it can be a bit flakey. Have you tried creating a wrapper component for `<vue-json-pretty>` that overrides this `word-wrap` style?